examples/example-vite-kitchen-sink/src/app/layout.tsx

```
@@ -0,0 +1,22 @@
+import { TooltipProvider } from "@/components/ui/tooltip";
+
+import Sidebar from "@/components/sidebar";
+import Header from "@/components/header";
+import StarknetProvider from "@/components/starknet-provider";
+
+
+export default function RootLayout({ children }: React.PropsWithChildren<{}>) {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)